tests/multigrid_invert_test.cpp

```diff
@@ -130,6 +130,7 @@ void setEigParam(QudaEigParam &mg_eig_param, int level)
   mg_eig_param.nEv = mg_eig_nEv[level];
   mg_eig_param.nKr = mg_eig_nKr[level];
   mg_eig_param.nConv = nvec[level];
+  mg_eig_param.batched_rotate = mg_eig_batched_rotate[level];
```

Comment:
This should be added to `multigrid_evolve_test.cpp` for completeness.

Reply:
https://github.com/lattice/quda/commit/4bb34ceb48acf7061377d2ef5bce24f33ed11fa2